CONTRIBUTING.md

```diff
@@ -1,59 +1,58 @@
-# Contributing guidelines
-
-We thank you in advance :thumbsup: :tada: for taking the time to contribute, whether with *code* or with *ideas*, to the NeIC SDA project.
+# Contributing Guidelines
 
+We thank you in advance 👍 🎉 for taking the time to contribute, whether with *code* or with *ideas*, to the NeIC SDA project.
 
 ## Did you find a bug?
 
-* Ensure that the bug was not already reported by [searching under Issues].
-
-* If you're unable to find an (open) issue addressing the problem, [open a new one]. Be sure to prefix the issue title with **[BUG]** and to include:
+- Ensure that the bug was not already reported by [searching under Issues].
 
-  - a *clear* description,
-  - as much relevant information as possible, and
-  - a *code sample* or an (executable) *test case* demonstrating the expected behaviour that is not occurring.
+- If you're unable to find an (open) issue addressing the problem, open a new one by using the following [template to report a bug]. Be sure to include:
 
-* If possible, use the following [template to report a bug].
+      - a *clear* description,
+      - as much relevant information as possible, and
+      - a *code sample* or an (executable) *test case* demonstrating the expected behaviour that is not occurring.
 
+- If possible, prefix the issue title with a short identifier for the relevant repository component, e.g. **[ingest]**, **[postgres]** etc.
 
 ## How to work on a new feature/bug
 
-Create an issue on Github or you can alternatively pick one already created.
+- Create an issue on Github or you can alternatively pick one already created.
 
-Assign yourself to that issue.
+- Assign yourself to that issue.
 
-Discussions on how to proceed about that issue take place in the comment section on that issue.
+- Discussions on how to proceed about that issue take place in the comment section on that issue.
 
-Some of the work might have been done already by somebody, hence we avoid unnecessary work duplication and a waste of time and effort. Other reason for discussing the issue beforehand is to communicate with the team the changes as some of the features might impact different components, and we can plan accordingly.
+Some of the work might have been done already by a co-worker. In this case, unnecessary work duplication and waste of time and effort are to be avoided. As some of the features might impact different components, it is a good idea to discuss the issue beforehand with the team and communicate the intended changes so that planning can be done accordingly.
```

Comment:
```suggestion
To avoid unnecessary work duplication and waste of time and effort, it's generally a good idea to discuss the issue beforehand with team. Some of the work might have been done already by a co-worker. Also, as some of the features might impact different components, please communicate the intended changes, so that planning can be done accordingly.
```